.eslintrc

```diff
@@ -3,8 +3,8 @@
         "node": true
     },
 
-    "ecmaFeatures": {
-        "globalReturn": false
+    "parserOptions": {
+        "ecmaVersion": 6,
```

Comment:
Currently this project still uses ES5. Also I really want to keep `globalReturn` disabled.